codegen/template.go

```diff
@@ -81,9 +99,17 @@ type Template struct {
 
 // NewTemplate creates a bundle of templates.
 func NewTemplate() (*Template, error) {
```

Comment:
Will find/replace `NewTemplate` break any user of zanzibar? I think users don't need to call `NewTemplate` directly but call module system.